client/my-sites/email/titan-set-up-thank-you/index.tsx

```diff
@@ -86,7 +97,18 @@ const TitanSetUpThankYou = ( props: TitanSetUpThankYouProps ): JSX.Element => {
 					'Add or delete mailboxes, migrate existing emails, configure a catch-all email, and much more.'
 				),
 				stepCta: (
-					<FullWidthButton href={ emailManagementPath }>{ translate( 'Manage' ) }</FullWidthButton>
+					<FullWidthButton
+						href={ emailManagementPath }
+						onClick={ () => {
+							recordEmailActionTrackEvent( {
+								provider: 'titan',
+								app: 'manage',
+								context: 'checkout-thank-you',
+							} );
+						} }
+					>
+						{ translate( 'Manage' ) }
+					</FullWidthButton>
```

Comment:
I don't think we want this click to trigger this event -- the "Manage" button is internal and is not launching an external email app.

Reply:
Done